<ui version="4.0" >
 <class>TowaryLista</class>
 <widget class="QDialog" name="TowaryLista" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>299</width>
    <height>352</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Wybierz towar z listy</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1" >
       <property name="maximumSize" >
        <size>
         <width>50</width>
         <height>32767</height>
        </size>
       </property>
       <property name="text" >
        <string>Typ:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox1" >
       <item>
        <property name="text" >
         <string>Towar</string>
        </property>
       </item>
       <item>
        <property name="text" >
         <string>Usługa</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="Q3ListView" name="listView1" >
     <property name="selectionMode" >
      <enum>Q3ListView::Single</enum>
     </property>
     <property name="allColumnsShowFocus" >
      <bool>true</bool>
     </property>
     <property name="resizeMode" >
      <enum>Q3ListView::LastColumn</enum>
     </property>
     <column>
      <property name="text" >
       <string>Index</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Nazwa</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel4" >
       <property name="text" >
        <string>Cena jedn.:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox2" >
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="maximum" >
        <number>4</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="cenaEdit" >
       <property name="enabled" >
        <bool>true</bool>
       </property>
       <property name="inputMask" >
        <string>99999999,99; </string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
       <property name="readOnly" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="textLabel1_2" >
       <property name="text" >
        <string>Ilość:</string>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="countEdit" >
       <property name="inputMask" >
        <string>99999999; </string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>131</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="textLabel3_2" >
           <property name="text" >
            <string>Netto:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nettoLabel" >
           <property name="minimumSize" >
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>0,00</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <item>
          <widget class="QLabel" name="textLabel3" >
           <property name="text" >
            <string>Brutto:</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="bruttoLabel" >
           <property name="minimumSize" >
            <size>
             <width>80</width>
             <height>0</height>
            </size>
           </property>
           <property name="text" >
            <string>0,00</string>
           </property>
           <property name="wordWrap" >
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="6" column="0" >
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>43</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okBtn" >
       <property name="text" >
        <string>O&amp;K</string>
       </property>
       <property name="shortcut" >
        <string>Alt+K</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>43</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelBtn" >
       <property name="text" >
        <string>Can&amp;cel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>43</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="5" column="0" >
    <widget class="Line" name="line3" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>okBtn</sender>
   <signal>clicked()</signal>
   <receiver>TowaryLista</receiver>
   <slot>doAccept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>TowaryLista</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox1</sender>
   <signal>activated(int)</signal>
   <receiver>TowaryLista</receiver>
   <slot>comboBox1Changed(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listView1</sender>
   <signal>selectionChanged(Q3ListViewItem*)</signal>
   <receiver>TowaryLista</receiver>
   <slot>lv1selChanged(Q3ListViewItem*)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>TowaryLista</receiver>
   <slot>spinChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>countEdit</sender>
   <signal>selectionChanged()</signal>
   <receiver>TowaryLista</receiver>
   <slot>calcNetto()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>countEdit</sender>
   <signal>lostFocus()</signal>
   <receiver>TowaryLista</receiver>
   <slot>calcNetto()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>countEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>TowaryLista</receiver>
   <slot>calcNetto()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
